5_analyze_data/notebooks/UMAP_analysis/scripts/umap_num_sc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create 2D UMAP using number of single cells
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+import seaborn as sns
+from matplotlib.colors import rgb2hex
+from mpl_toolkits.mplot3d import axes3d, Axes3D
+import umap
+
+import UMAPutils as utils
```

Comment:
Consider removing these imports if they are not used.

Reply:
Fixed for all notebooks